pkg/hubble/peer/handler.go

```diff
@@ -37,6 +37,10 @@ func newHandler(withoutTLSInfo bool, addressPref serviceoption.AddressFamilyPref
 	}
 }
 
+func (h *handler) Name() string {
+	return "hubble-peers"
```

Comment:
Nit: should this be singular? Package is singular, and we refer to it as the "hubble peer service" elsewhere.

Reply:
🤔 did this change?